service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -83,6 +87,8 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(
                 }
             } catch (FeeRegisterException ignore) {
                 data.put("displayAmountToPay", NO.getValue());
+                applicationEventPublisher.publishEvent(new FailedPBAPaymentEvent(callbackRequest, authorization, userId,
+                    C110A_APPLICATION));
```

Comment:
I think this should be removed and notification should be send only in submitted event. Otherwise we'll be sending notification every time we go to case submission event, and we want to do it only once for each case after it's submitted.